go.mod

```diff
@@ -82,6 +82,7 @@ require (
 	kubevirt.io/controller-lifecycle-operator-sdk/api v0.0.0-20220329064328-f3cc58c6ed90
 	kubevirt.io/qe-tools v0.1.8
 	libvirt.org/go/libvirt v1.9004.0
+	libvirt.org/go/libvirtxml v1.9008.0
```

Comment:
it would be neat to use `LIBVIRT_VERSION` from `hack/rpm-deps.sh:9`
I think we should correct the version to the same as the rpm. Should not be a big problem though, for this PR.

Reply:
FWIW, you should pick the `libvirtxml` XML version that matches the version of the libvirt *daemon* (not libvirt go api) that you are going to run ``kubevirt`` against. This ensures that ``libvirtxml`` knows how to parse every aspect of the schema that the libvirt daemon can potentially spit out, and thus guarantee you can round-trip `` xml -> parse -> obj ->  format -> xml``  without risk of loosing data.

Reply:
```
libvirt.org/go/libvirt v1.9004.0
libvirt.org/go/libvirtxml v1.9008.0
```
I would have preferred the versions to have been the same :slightly_smiling_face:

Reply:
@fossedihelm I left the latest tag because of the above discussion